src/wwwroot/css/main.css

```diff
@@ -0,0 +1,3 @@
eft:0.9em}@media (min-width: 641px){.breadcrumbs li{font-size:16px;line-height:1.25}}@media only screen and (-webkit-min-device-pixel-ratio: 2), only screen and (min--moz-device-pixel-ratio: 2), only screen and (-o-min-device-pixel-ratio: 20 / 10), only screen and (min-device-pixel-ratio: 2), only screen and (min-resolution: 192dpi), only screen and (min-resolution: 2dppx){.breadcrumbs li{background-image:image-url("separator-2x.png");background-size:6px 11px}}.breadcrumbs li:first-child{background-image:none;margin-left:0;padding-left:0}.breadcrumbs a{color:#0b0c0c}.phase-banner{padding:10px 0 8px;border-bottom:1px solid #bfc1c3}@media (min-width: 641px){.phase-banner{padding-bottom:10px}}.phase-banner p{display:table;margin:0;color:#000;font-family:"nta",Arial,sans-serif;font-weight:400;text-transform:none;font-size:14px;line-height:1.1428571429}@media (min-width: 641px){.phase-banner p{font-size:16px;line-height:1.25}}.phase-banner .phase-tag{display:-moz-inline-stack;display:inline-block;margin:0 8px 0 0;padding:2px 5px 0;font-family:"nta",Arial,sans-serif;font-weight:700;text-transform:none;font-size:14px;line-height:1.1428571429;text-transform:uppercase;letter-spacing:1px;text-decoration:none;color:#fff;background-color:#005ea5}@media (min-width: 641px){.phase-banner .phase-tag{font-size:16px;line-height:1.25}}.phase-banner span{display:table-cell;vertical-align:baseline}.phase-banner-alpha,.phase-banner-beta{padding:10px 0 8px;border-bottom:1px solid #bfc1c3}@media (min-width: 641px){.phase-banner-alpha,.phase-banner-beta{padding-bottom:10px}}.phase-banner-alpha p,.phase-banner-beta p{display:table;margin:0;color:#000;font-family:"nta",Arial,sans-serif;font-weight:400;text-transform:none;font-size:14px;line-height:1.1428571429}@media (min-width: 641px){.phase-banner-alpha p,.phase-banner-beta p{font-size:16px;line-height:1.25}}.phase-banner-alpha .phase-tag,.phase-banner-beta .phase-tag{display:-moz-inline-stack;display:inline-block;margin:0 8px 0 0;padding:2px 5px 0;font-family:"nta",Arial,sans-serif;font-weight:700;text-transform:none;font-size:14px;line-height:1.1428571429;text-transform:uppercase;letter-spacing:1px;text-decoration:none;color:#fff;background-color:#005ea5}@media (min-width: 641px){.phase-banner-alpha .phase-tag,.phase-banner-beta .phase-tag{font-size:16px;line-height:1.25}}.phase-banner-alpha span,.phase-banner-beta span{display:table-cell;vertical-align:baseline}.phase-tag{display:-moz-inline-stack;display:inline-block;margin:0 8px 0 0;padding:2px 5px 0;font-family:"nta",Arial,sans-serif;font-weight:700;text-transform:none;font-size:14px;line-height:1.1428571429;text-transform:uppercase;letter-spacing:1px;text-decoration:none;color:#fff;background-color:#005ea5}@media (min-width: 641px){.phase-tag{font-size:16px;line-height:1.25}}.govuk-box-highlight{margin:1em 0;padding:2em 1em;color:#fff;background:#28a197;text-align:center}@-moz-document regexp('.*'){details summary:not([tabindex]){display:list-item;display:revert}}
```

Comment:
Should the built files be committed?

Reply:
Added these for convenience. ie the npm network may be down or the localhost may not have the appropriate setup ie c++.